### PR TITLE
Add unumpy feedstock.

### PR DESCRIPTION
**Pull request by hameerabbasi**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hameerabbasi** [comment]
cc: @conda-forge/help-python Is the linting correct? I've followed the example exactly.

**Comment by hameerabbasi** [comment]
And the docs say the same thing

File: recipes/unumpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  sha256: 07777c96c0d559b27739f64f91c286842fc99ae409df61bd0d9f3c49c2a4c4f6
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```
Comment (xhochy) [inline recipes/unumpy/meta.yaml]: You can use `python >=3.5` to mark this as being Python 3.5 only.

```suggestion
    - python >=3.5
```

**Comment by hameerabbasi** [comment]
Is there a way to build wheels from conda-forge yet?

**Comment by xhochy** [comment]
> Is there a way to build wheels from conda-forge yet?

No, not yet.

File: recipes/unumpy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python
```
Comment (xhochy) [inline recipes/unumpy/meta.yaml]: You also need this in run
```suggestion
    - python >=3.5
```

**Comment by hameerabbasi** [comment]
I think this is ready for a review/merge. Thanks for the help!

**Pull request merged**